### PR TITLE
More hair options

### PR DESCRIPTION
**Pull request by rhicora**
Adds 5 new hair options.

image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 #Ignore byond config folder.
 /cfg/**/*
+/config/admins.txt
```
Comment (AndrewL97) [inline .gitignore]: ahhhhh

**Comment by AndrewL97** [comment]
damn that was fast

Are all the new hair options available in character creation? @rhicora

**Comment by rhicora** [comment]
You bet.

File: .gitignore (hunk 1 above)
Comment (rhicora) [reply .gitignore]: This file shouldn't be committed, it is individual for whoever is hosting a test server. The one in the repo works for you but doesn't work for everyone else.

**Pull request merged**